### PR TITLE
test_battery: add new field ProjectID to schema (Redmine 10510) 

### PR DESCRIPTION
**Pull request by gluneau**
CCNA requirement

### CODE REVIEW

File: SQL/Archive/17.0/2016-07-08-AddProjectIDTotest_batteryTable.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE test_battery ADD `ProjectID` int(11) DEFAULT NULL;
```
Comment (Jkat) [inline SQL/Archive/17.0/2016-07-08-AddProjectIDTotest_batteryTable.sql]: Add "AFTER Stage" ?

Comment (gluneau) [inline SQL/Archive/17.0/2016-07-08-AddProjectIDTotest_batteryTable.sql]: Yes, but most importantly BEFORE SubprojectID and... look at this kitten:

image

**Comment by xlecours** [comment]
Please add a foreign key to the Project table (and keep the default null for studies that don't use Projects)

**Comment by driusan** [comment]
Wasn't the plan to make projects required and just use a default project name like "LORIS" for projects that don't want to use it for 17.0?

**Comment by driusan** [comment]
@gluneau From meeting: This is missing the code to use this field in the NDB_BVL_Battery(?).

**Comment by gluneau** [comment]
Closing until @mohadesz and CCNA needs it again.

**Pull request closed without merging**